Source/ObjectiveDropboxOfficial/Shared/Handwritten/OAuth/DBOAuthResult.m

```diff
@@ -18,14 +18,18 @@ + (DBOAuthErrorType)getErrorType:(NSString *)errorType {
   static dispatch_once_t onceToken;
   dispatch_once(&onceToken, ^{
     errorTypeLookup = @{
-      @"unauthorized_client" : [NSNumber numberWithInt:DBAuthUnauthorizedClient],
-      @"access_denied" : [NSNumber numberWithInt:DBAuthAccessDenied],
-      @"unsupported_response_type" : [NSNumber numberWithInt:DBAuthUnsupportedResponseType],
-      @"invalid_scope" : [NSNumber numberWithInt:DBAuthInvalidScope],
-      @"server_error" : [NSNumber numberWithInt:DBAuthServerError],
-      @"temporarily_unavailable" : [NSNumber numberWithInt:DBAuthTemporarilyUnavailable],
-      @"inconsistent_state" : [NSNumber numberWithInt:DBAuthInconsistentState],
-      @"" : [NSNumber numberWithInt:DBAuthUnknown],
+      @"unauthorized_client" : @(DBAuthUnauthorizedClient),
+      @"access_denied" : @(DBAuthAccessDenied),
+      @"unsupported_response_type" : @(DBAuthUnsupportedResponseType),
+      @"invalid_scope" : @(DBAuthInvalidScope),
+      @"server_error" : @(DBAuthServerError),
+      @"temporarily_unavailable" : @(DBAuthTemporarilyUnavailable),
+      @"invalid_request": @(DBAuthInvalidRequest),
+      @"invalid_client": @(DBAuthInvalidClient),
+      @"invalid_grant": @(DBAuthInvalidGrant),
+      @"unsupported_grant_type": @(DBAuthUnsupportedGrantType),
+      @"inconsistent_state" : @(DBAuthInconsistentState),
+      @"" : @(DBAuthUnknown),
```

Comment:
hmmm, it seems odd that we have an empty string mapped to an error type... We should probably fix this at some point. This is just confusing. Maybe add a todo.

Reply:
yeah, it's confusing. I think it's pretty safe to delete this entry because anything that's not a valid key would be mapped to unknown anyways.

Reply:
👍